src/schema/metadata/B0FieldIdentifier.yaml

```
@@ -0,0 +1,13 @@
+---
+name: B0FieldIdentifier
+description: |
+  The presence of this key states that this particular 3D or 4D image MAY be
+  used for fieldmap estimation purposes.
+  The `B0FieldIdentifier` MUST be a unique string within one participant's tree,
+  shared only by the images meant to be used as inputs for the estimation of a
+  particular instance of the *B<sub>0</sub> field* estimation.
+  It is RECOMMENDED to derive this identifier from DICOM Tags, for example,
+  the *Protocol Name* `(0018, 1030)` or the *Sequence Name* `(0018, 0024)`
+  when the former is not defined (for example, in GE devices.)
```

Here we refer to DICOM tags with brackets again (inconsistent with the rest  of the spec). Apparently we missed that in a previous PR with the regexps.

I suggest to change this to something like:

```suggestion
  DICOM tag 0018, 1030 `Protocol Name`, or DICOM tag 0018, 0024 `Sequence Name`
  when the former is not defined (for example, in GE devices.)
```